demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo.csproj

```diff
@@ -6,6 +6,13 @@
     <UserSecretsId>7e7187fe-8562-4710-93fe-40f33e289558</UserSecretsId>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
What's the function of this??

Reply:
It was created by visual Studio, I'll delete them as those files no longer exist
.